Makefile

```diff
@@ -126,9 +126,7 @@ PR_COMMIT := $(shell echo $$CLONEREFS_OPTIONS | jq '.refs[0].pulls[0].sha')
 PULL_NUMBER := $(shell echo $$CLONEREFS_OPTIONS | jq '.refs[0].pulls[0].number')
 
 ## -- Static code analysis (lint) targets --
-
 .PHONY: lint
-## Runs linters on Go code files and YAML files - DISABLED TEMPORARILY
```

Comment:
@akram I think this commend is being used over `help` target. Could you try `make help` and see the results?

Reply:
I think only "DISABLED TEMPORARILY" should have been removed.